utilities/base64Handler.js

```diff
@@ -11,6 +11,29 @@ class Base64Handler {
             return filepath;
         });
     }
+
+    static async toImageFromReq (req, res, next) {
+        const { profilepicture } = req.body;
+
+        if (profilepicture) {
+            const { host } = req.headers;
+
+            let { str, image_ext, filename, path } = profilepicture;
+            const format = `data:image/${image_ext};base64,` // format
+            path = `public/${req.body.email}/profile_pic/`
+            const formattedString = `${format}${str}`
+
+            base64Img.img(formattedString, path, filename, (err, filepath) => {
+                if (err) throw err;
+                // attach decoded saved image to request body
+                req.body.profilepicture = `${host}${filepath.substr(6, filepath.length)}`;
+
+                next();
+            });
+        } else {
+            next();
+        }
+    }
```

Comment:
I feel we should use formData to handle file uploads and upload to a cloud server(Cloudinary for example) Then just save the URL in the database. This lifts more work from the server and also saves a lot of storage space in the database. Don't worry about it though, i'll handle it. Thanks